uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenDropInServiceRepository.kt

```diff
@@ -0,0 +1,67 @@
+package com.karhoo.uisdk.screen.booking.booking.payment.adyen
+
+import android.content.Context
+import androidx.core.app.JobIntentService
+
+class AdyenDropInServiceRepository(val context: Context) : AdyenDropInServiceMVP.Repository {
```

Comment:
Do we need a test class for this?

Reply:
I was thinking about it, but not sure how much value it would add. as we need to mock the context and shared preferences .. so storing and retrieving would be mostly based on what we mock. There is no special business logic here

Reply:
Was just thinking about the call to clear if value is null.